web/tls_config.go

```diff
@@ -35,6 +35,8 @@ var (
 	errNoTLSConfig = errors.New("TLS config is not present")
 )
 
+const ErrNoListeners = errors.New("no web listen address or systemd socket flag specified")
```

Comment:
Huh, I didn't know that this was a valid expression. Is this a new Go feature?

Reply:
Looks like it's still not valid.

```
web/tls_config.go:38:24: errors.New("no web listen address or systemd socket flag specified") (value of type error) is not constant
```

Reply:
I just mashed it out in my IDE but didn't actually test it. I was running go 1.20 so it's possible it's a new thing.

I'll change it to a Var tomorrow.

Reply:
Should be good to go now @SuperQ